README.adoc

```diff
@@ -297,15 +297,15 @@ service_filter_label_name: VALUE
 |The URL used to access and query the Prometheus Server.
 [source,yaml]
 ----
-products:
+components:
```

Comment:
If we finish up with ExternalServices I guess it should be updated in the doc as well.